config/initializers/ab_tests.rb

```diff
@@ -116,4 +116,15 @@ def self.all
     buckets: { desktop_ft_unlock_option_shown:
         IdentityConfig.store.desktop_ft_unlock_setup_option_percent_tested },
   ).freeze
+
+  DOC_AUTH_PASSPORT = AbTest.new(
+    experiment_name: 'Passport allowed',
+    should_log: /^idv/i,
+    buckets: {
+      passport_allowed: IdentityConfig.store.doc_auth_passports_enabled ?
+        IdentityConfig.store.doc_auth_passports_percent : 0,
+    },
+  ) do |service_provider:, session:, user:, user_session:, **|
+    user&.uuid
```

Comment:
Comparing with the RECAPTCHA_SIGN_IN proc, it looks like this could be:

```suggestion
  ) do |user:, **|
    user&.uuid
```